resources/sharedconfig.ini

```diff
@@ -65,4 +65,15 @@ version_minor=64
 
 # The port the chat server is started and listening on
 # Used in chat and world servers
-chat_server_port=2005
\ No newline at end of file
+chat_server_port=2005
+
+sqlite_database_path=resServer/dlu.sqlite
+
+# If using sqlite, set this to 1 and the others to 0
+using_sqlite=1
```

Comment:
Since it's either/or, maybe this should be an enum, i.e.

```ini
database_kind=sqlite
# or
database_kind=mysql
```

Reply:
Yeah, it would be worth it to be quite liberal in what we accept here and trim, lowercase it before comparing, accepting mariadb (and possibly maria) as an alias for mysql and maybe even just picking sqlite if unrecognizable. But I do think it's the better design.

Reply:
I think if the value is unrecognizable, we need to default to mariadb since thats what it was before unless you can think of a better way to detect if someone was using mariadb prior and has updated